big_o_exercise/readme.md

```diff
@@ -4,16 +4,26 @@
 
 Simplify the following big O expressions as much as possible:
 
-1. `O(n + 10)`
+1. `O(n + 10)`  
+// O(n)
 2. `O(100 * n)`
-3. `O(25)`
+// O(n)
+3. `O(1)`
+// O(n)
 4. `O(n^2 + n^3)`
+// O(n^3)
 5. `O(n + n + n + n)`
+// O(n)
 6. `O(1000 * log(n) + n)`
+// log(n)
 7. `O(1000 * n * log(n) + n)`
+// nlog(n)
 8. `O(2^n + n^2)`
```

Comment:
`2^n` grows faster than `n^2`, but we didn't talk about this in class so I wouldn't sweat it.